packages/ibm-products-styles/src/components/DataSpreadsheet/_data-spreadsheet.scss

```diff
@@ -267,10 +267,6 @@ $header-cell-background: $layer-accent-01;
     width: $spacing-01;
     background-color: $background-brand;
   }
-  .#{$block-class}__th--active-header,
-  .#{$block-class}__td-th--active-header.#{$block-class}__td {
-    background-color: $background-selected-hover;
-  }
```

Comment:
any reason why we remove this style? its not highlighting the cell coordinates like before.
screenshot from production
<img width="1623" alt="Screenshot 2024-06-18 at 11 31 56 AM" src="https://github.com/carbon-design-system/ibm-products/assets/47176249/fd654e39-197e-4228-9d29-86bc6b28a660">

Reply:
Yes, this class is the reason this bleed-through is occurring

![Screenshot 2024-06-17 at 11 17 56 PM](https://github.com/carbon-design-system/ibm-products/assets/10100397/ff2dc099-e14e-4c0d-8729-a317263a8004)

Reply:
adding this background to cell container might solve this issue, could you check it once?
<img width="2168" alt="Screenshot 2024-06-18 at 12 13 33 PM" src="https://github.com/carbon-design-system/ibm-products/assets/47176249/83251c53-d3ef-48fe-babc-bf0e94b5ae48">

Reply:
we can apply default background color to the parent, as in screenshot, so we can avoid compromising on this feature?

<img width="2168" alt="Screenshot 2024-06-18 at 12 31 06 PM" src="https://github.com/carbon-design-system/ibm-products/assets/47176249/25863236-baaf-4acf-872c-752c4a963c45">

Reply:
Thanks, I will take a look later today

Reply:
I have fixed this, thanks!

![Screenshot 2024-06-18 at 8 52 46 AM](https://github.com/carbon-design-system/ibm-products/assets/10100397/ca3902ea-da26-45ad-a2df-fc79e54c594b)